astropy/nddata/ccddata.py

```diff
@@ -270,18 +285,19 @@ def uncertainty(self, value):
 
     def to_hdu(self, hdu_mask='MASK', hdu_uncertainty='UNCERT',
                hdu_flags=None, wcs_relax=True,
-               key_uncertainty_type='UTYPE', as_image_hdu=False):
+               key_uncertainty_type='UTYPE', as_image_hdu=False,
+               hdu_psf="PSFIMAGE"):
         """Creates an HDUList object from a CCDData object.
 
         Parameters
         ----------
-        hdu_mask, hdu_uncertainty, hdu_flags : str or None, optional
+        hdu_mask, hdu_uncertainty, hdu_flags, hdu_psf: str or None, optional
             If it is a string append this attribute to the HDUList as
             `~astropy.io.fits.ImageHDU` with the string as extension name.
             Flags are not supported at this time. If ``None`` this attribute
             is not appended.
-            Default is ``'MASK'`` for mask, ``'UNCERT'`` for uncertainty and
-            ``None`` for flags.
+            Default is ``'MASK'`` for mask, ``'UNCERT'`` for uncertainty,
+            "PSFIMAGE" for psf, and ``None`` for flags.
```

Comment:
```suggestion
            ``'PSFIMAGE'`` for psf, and ``None`` for flags.
```
Was this formatted so differently on purpose? The single/double quotes in the function signature above are apparently homogenised automatically in the processes docs, but the way this is rendered in the docstring https://astropy--13743.org.readthedocs.build/en/13743/api/astropy.nddata.CCDData.html#astropy.nddata.CCDData.to_hdu
looks very strange to read.